csrc/host_ir/host_ir.cpp

```diff
@@ -237,6 +237,56 @@ bool Synchronize::sameAs(const Statement* other) const {
   return false;
 }
 
+StartCoalescing::StartCoalescing(IrBuilderPasskey passkey) : Expr(passkey) {
+  NVF_ERROR(passkey.ir_container_ != nullptr);
+  NVF_ERROR(
+      passkey.ir_container_->isA<HostIrContainer>(),
+      this,
+      "must be registered in a HostIrContainer");
+}
+
+NVFUSER_DEFINE_CLONE_AND_CREATE(StartCoalescing)
+
+std::string StartCoalescing::toString(int indent_size) const {
+  std::stringstream ss;
+  indent(ss, indent_size) << "StartCoalescing" << std::endl;
+  return ss.str();
+}
+
+std::string StartCoalescing::toInlineString(int indent_size) const {
+  NVF_CHECK(false, "Cannot be printed inline");
+}
+
+// TODO: implement
```

Comment:
Doesn't [the default one](https://github.com/NVIDIA/Fuser/blob/2c8a7c657857912c1fefe821ee37d3b18118b8f1/csrc/ir/base_nodes.cpp#L331) just work?

Reply:
You are right, I removed those methods.

BTW, should we consider adding comparison of attributes in the default sameAs? If yes, we could probably remove a lot of overriding definitions

Reply:
Check sameOp. I believe that compares attributes but I'm afk to double check